test/integration/integration_test.go

```diff
@@ -93,9 +95,10 @@ func testBasicBaseConfig(grpcPort, perSecond string, local_cache_size string) fu
 
 		local_cache_size_val, _ := strconv.Atoi(local_cache_size)
 		enable_local_cache := local_cache_size_val > 0
+		store := stats.NewDefaultStore()
 
 		go func() {
-			runner.Run()
+			runner.Run(store)
```

Comment:
I think it would be a bit cleaner to do something like:
1) Allocate a runner object
2) Have a method to get the store
3) Then have the run method which is a method on the object and not static.

This would avoid special casing and allow the test code to grab the store.

Reply:
thanks for the suggestion. Just changed it, it looks better.